app/lib/features/chat_ng/widgets/events/chat_event_item.dart

```diff
@@ -94,20 +96,29 @@ class ChatEventItem extends StatelessWidget {
     String messageId,
     RoomEventItem item,
     bool isUser,
-    bool showAvatar,
+    bool nextMessageGroup,
```

Comment:
what is this flag supposed to indicate? it doesn't make sense to me from its name...

Reply:
This is a check whether the next message comes as group from same user. The flag term is taken from ```flutter_chat_ui``` => [isNextMessageInGroup](https://pub.dev/documentation/flutter_chat_ui/latest/flutter_chat_ui/Message/bubbleBuilder.html)